lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -142,12 +156,16 @@ bench( pkg+'::set:s', function benchmark( bm ) {
 	mu = 2.0;
 	s = 3.0;
 	dist = new Logistic( mu, s );
+	len = 100;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = uniform( EPS, 100.0 );
+	}
 
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		y = ( 100.0*randu() ) + EPS;
-		dist.s = y;
-		if ( dist.s !== y ) {
+		dist.beta = y[ i % len ];
+		if ( dist.beta !== y[ i % len ] ) {
```

Comment:
```suggestion
		dist.s = y[ i % len ];
		if ( dist.s !== y[ i % len ] ) {
```

Same comment as above